### DISCUSSION TITLE
Managed Instance is not available for the chosen subscription and region.

### DISCUSSION THREAD

**Discussion by xluciola** [AzureCommunityPL/group#139] [Pomoc]
czy możliwe jest , że brakuje zasobów na SQl managed instance .
mam wersję próbną i 170 $ na wydatki a nie mogę utworzyć tego zasobu.

dziękuję

**Comment by eklime** [comment]
Dokumentacji się zapewne nie przeczytało?
https://docs.microsoft.com/en-us/azure/azure-sql/managed-instance/resource-limits?WT.mc_id=AZ-MVP-5001765#supported-subscription-types

**Comment by xluciola** [comment]
przeczytało się dopiero teraz :)
skoro mam subskrypcję testową myślałem, że mogę spróbować wszystkiego,
chciałem przetestować i sprawdzić czy program ERP którego używam poradzi sobie z taką bazą i jakie będzie generował koszty i czy lepiej skorzystać z takiej bazy czy utworzyć sobie ja np na dedyku,
